pom.xml

```diff
@@ -18,51 +18,51 @@
   <modules>
     <module>bom</module>
     <module>parent</module>
-    <module>atomix</module>
-    <module>auth</module>
-    <module>broker</module>
-    <module>broker-client</module>
-    <module>protocol-test-util</module>
     <module>dist</module>
-    <module>protocol</module>
-    <module>bpmn-model</module>
-    <module>logstreams</module>
-    <module>stream-platform</module>
-    <module>engine</module>
-    <module>transport</module>
-    <module>util</module>
-    <module>test-util</module>
-    <module>msgpack-core</module>
-    <module>msgpack-value</module>
     <module>build-tools</module>
-    <module>gateway-protocol</module>
-    <module>gateway-protocol-impl</module>
-    <module>gateway</module>
-    <module>gateway-rest</module>
-    <module>clients/java</module>
-    <module>exporter-api</module>
-    <module>exporter-test</module>
-    <module>protocol-asserts</module>
-    <module>exporters/elasticsearch-exporter</module>
-    <module>exporters/opensearch-exporter</module>
-    <module>protocol-impl</module>
-    <module>protocol-jackson</module>
-    <module>zb-db</module>
-    <module>expression-language</module>
-    <module>feel</module>
-    <module>dmn</module>
-    <module>snapshot</module>
-    <module>journal</module>
-    <module>qa</module>
-    <module>benchmarks/project</module>
```

Comment:
❓ Did we include Operate already in the benchmarks? If so, why not leave it in the root?

Reply:
That is true, but only partly. We need to be aware that there also is benchmark project that includes related to the running applications. In general the benchmarks are mostly targeting and used by Zeebe. I would keep it for now in our space.